checks/raw/security_policy.go

```diff
@@ -112,3 +137,87 @@ func isSecurityPolicyFilename(name string) bool {
 		strings.EqualFold(name, "doc/security.rst") ||
 		strings.EqualFold(name, "docs/security.rst")
 }
+
+var checkSecurityPolicyFileContent fileparser.DoWhileTrueOnFileContent = func(path string, content []byte,
+	args ...interface{},
+) (bool, error) {
+	if len(args) != 1 {
+		return false, fmt.Errorf(
+			"checkSecurityPolicyFileContent requires exactly one argument: %w", errInvalidArgLength)
+	}
+	pfiles, ok := args[0].(*[]checker.File)
+	if !ok {
+		return false, fmt.Errorf(
+			"checkSecurityPolicyFileContent requires argument of type *[]checker.File: %w", errInvalidArgType)
+	}
+
+	if len(content) == 0 {
+		// perhaps there are more policy files somewhere else,
+		// keep looking (true)
+		return true, nil
+	}
+
+	// TODO: there is an assertion here that if content has length
+	//       then (*pfiles)[0] is not nil (can that be checked)
+	//       (so is this test necessary?)
+	if (*pfiles) != nil {
+		// preserve file type
+		tempType := (*pfiles)[0].Type
+		linkedContentLen, urls, emails, discvuls := countUniquePolicyHits(string(content))
+		contentMetrics := fmt.Sprintf("%d,%d,%d,%d,%d", len(content), linkedContentLen, urls, emails, discvuls)
+		(*pfiles)[0] = checker.File{
+			Path:    path,
+			Type:    tempType,
+			Offset:  checker.OffsetDefault,
+			Snippet: contentMetrics,
+		}
+	}
+
+	// stop here found something, no need to look further (false)
+	return false, nil
+}
+
+func countUniquePolicyHits(policyContent string) (int, int, int, int) {
+	var urls, strlenOfUrls, emails, strlenOfEmails, discvuls int
```

Comment:
Let's record the lines of the hit, so that we can add them later to details via `Info()`

Reply:
Check... got it, will do!

Reply:
Do, I did not do line hits, and I suppose it could be done.

Right now, I am using `regexp.FindAllString()` on the buffered content and not going "line by line". If this is a non-starter, I can go back and do finer grained parsing/telemetry to id the line (and col/offset per line) of the matches.

Your call...